File: containerSetup/submissions/3/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        long n = sc.nextLong();

        boolean prime = true;

        if (n < 2) {
            prime = false;
        } else {
            for (long i = 2; i * i <= n; i++) {
                if (n % i == 0) {
                    prime = false;
                    break;
                }
            }
        }

        System.out.println(prime ? "true" : "false");
    }
}

File: containerSetup/submissions/3/Main.cpp
#include <iostream>
using namespace std;

int main()
{
    long long n;
    cin >> n;

    bool prime = true;

    if (n < 2)
    {
        prime = false;
    }
    else
    {
        for (long long i = 2; i * i <= n; i++)
        {
            if (n % i == 0)
            {
                prime = false;
                break;
            }
        }
    }

    cout << (prime ? "true" : "false") << endl;

    return 0;
}